src/scales/scale.linear.js

```diff
@@ -12,6 +12,31 @@ module.exports = function(Chart) {
 	};
 
 	var LinearScale = Chart.LinearScaleBase.extend({
+
+		scaleLimitSnapshot: {
+			max: null,
+			min: null
+		},
+
+		/**
+		 * A util function to create snapshot of axis limits {max, min}
+		 * @Method chart.takeScaleLimitSnapshot
+		 * @param Integer {min} chart mininmum
```

Comment:
And you can also remove the @method which is useless (and actually wrong :) )

BTW, I'm going to setup jdoc as a dev dependency and add a gulp task to generate it (will do a pass on all the existing code comments to fix them).